bokehjs/src/lib/core/serialization/deserializer.ts

```diff
@@ -68,6 +68,7 @@ export class Deserializer {
     for (const instance of finalizable) {
       this.finalize?.(instance)
       instance.finalize()
+      instance.assert_initialized()
```

Comment:
This hits every property of every model, are there any appreciable performance implications? Would it make sense to put it behind a debug or elevated log level flag?

Reply:
All properties are fully initialized for all models, so this doesn't affect anything except correctness.

Reply:
It does a `prop.get_value()` on every property of every mode. That has some execution cost. I am asking about that cost.

Reply:
`get_value()` has no associated cost. In bokehjs value initialization is done upfront during initialization of models and their properties, and are relatively low cost. This is contrary to bokeh with complex logic and expensive initialization, and thus deferred initialization. In bokehjs any expensive computations are deferred to views and computed as needed (e.g. array materialization). Most of the cost of properties in bokehjs boils down to having an instance per property per model, which has non-negligible cost, but that's the cost to pay for having a rich reactive system built on top of this.